demo/api.md

```diff
@@ -803,7 +706,7 @@ Using the `lg` attribute will cause the size of the accordion trigger to render
 
 ### emphasis
 
-Using the `noToggleExpanded` attribute will allow for multiple `auro-accordion`'s in an `auro-accordion-group` to be open at the same time.
+Use the `emphasis` attribute to apply certain styles to the `auro-accordions`.
```

Comment:
This needs to be more descriptive, it's currently too vague.

Use the `emphasis` attribute to apply border highlights to the `auro-accordion` on hover and remove the chevron.